src/tasks/installers/olm.ts

```diff
@@ -279,4 +299,24 @@ export class OLMTasks {
       }
     }
   }
+
+  private createIndexCatalogSource(namespace: string, catalogSourceImage: string): CatalogSource {
```

Comment:
this function doesn't create anything but return CatalogSource yaml

Reply:
I renamed this method to `constructCatalogSource`